Client/Frontend/Home/JumpBackIn/JumpBackInViewModel.swift

```diff
@@ -145,6 +150,22 @@ class JumpBackInViewModel: FeatureFlaggable {
     private var hasJumpBackIn: Bool {
         return jumpBackInList.itemsToDisplay > 0
     }
+
+    private var isMultitasking: Bool {
+        guard let window = UIWindow.keyWindow else { return false }
+
+        return window.frame.width != window.screen.bounds.width && window.frame.width != window.screen.bounds.height
+    }
+
+    private func isPadInLandscapeSplit(split: CGFloat,
+                                       isPortrait: Bool = UIWindow.isPortrait,
+                                       device: UIUserInterfaceIdiom = UIDevice.current.userInterfaceIdiom) -> Bool {
+        guard device == .pad, !isPortrait, isMultitasking, let window = UIWindow.keyWindow
+        else { return false }
```

Comment:
```suggestion
        guard device == .pad, 
                   !isPortrait, 
                   isMultitasking, 
                   let window = UIWindow.keyWindow
        else { return false }
```

This is a nit. Feel free to ignore.

Reply:
GH HATES MY ALIGNMONSTS!